js/ai/src/generate.ts

```diff
@@ -22,7 +22,8 @@ import {
   z,
 } from '@genkit-ai/core';
 import { lookupAction } from '@genkit-ai/core/registry';
-import { toJsonSchema, validateSchema } from '@genkit-ai/core/schema';
+import { parseSchema, toJsonSchema } from '@genkit-ai/core/schema';
+import { StatusName } from '../../core/lib/statusTypes.js';
```

Comment:
this is bad import (relative cross package). Could you please make sure `StatusName` (and perhaps whatever else is in `js/core/src/statusTypes.ts`) is exported from `js/core/src/index.ts` and then here import it from `@genkit-ai/core`.